drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/referral_postings_loader.rb

```diff
@@ -146,7 +146,7 @@ def build_enrollment_rows
             personal_id: personal_id,
             entry_date: entry_date,
             household_id: household_id,
-            relationship_to_hoh: relationship_to_hoh(member_row),
+            relationship_to_hoh: row_value(member_row, field: 'RELATIONSHIP_TO_HOH_ID'),
```

Comment:
It was being set to a string value like `spouse_or_partner` which was being saved as `0` since it's an integer field